### PR TITLE
Remove old Discovery and SchemaRegistry specs

### PR DESCRIPTION
**Pull request by duglin**
Moves some old files into registry/todos as a reminder to update them

### CODE REVIEW

File: languages/languages-spec.md [hunk 1]
```diff
@@ -49,38 +49,38 @@ There are 4 types of status:
...
 ### Step 1. Open an issue to avoid redundant works
 
-Before starting to do any actual works, you should open an issue to explicitly declare 
+Before starting to do any actual works, you should open an issue to explicitly declare
 what you want to do.
 
 ### Step 2. Build the structure locally
 
-Currently, each spec directory(/cloudevents, /discovery, or /cesql, etc) 
+Currently, each spec directory(/cloudevents, /registry, or /cesql, etc)
```
Comment (jskeet) [inline languages/languages-spec.md:80]: Add a space before the bracket, while we're here?

Comment (duglin) [reply languages/languages-spec.md:80]: done

**Comment by matzew** [comment]
https://github.com/duglin/spec/blob/delDiscovery/registry/todos/primer.md#examples

should this being updated too?

**Comment by duglin** [comment]
> https://github.com/duglin/spec/blob/delDiscovery/registry/todos/primer.md#examples
> 
> should this being updated too?

Yes, that's why it's in the "todo" dir - at some point we'll move it out of that dir and we'll clean-up all of the text. I just didn't want to lose the doc and I like the nagging reminder to revisit it by having that dir.

**Comment by duglin** [comment]
Approved on the 2/23 call

**Pull request merged**